File: api/src/monitoring.py
import os
import sentry_sdk

if os.getenv("PRODUCTION", "False").lower() in ("true", "1", "t"):
    sentry_sdk.init(
        dsn="https://f17c56c681bd044ebea1ca1074eb837c@o555244.ingest.us.sentry.io/4507589807308800",
        traces_sample_rate=0.0,
        profiles_sample_rate=0.0,
    )

File: webapp/src/utils/monitoring.ts
if (import.meta.env.PROD && import.meta.env.SSR === false) {
  const Sentry = await import("@sentry/react");
  Sentry.init({
    dsn: "https://3b863adc28655f68ce8832a4f747668b@o555244.ingest.us.sentry.io/4507589829394432",
    integrations: [],
    replaysSessionSampleRate: 0.0,
    replaysOnErrorSampleRate: 1.0,
  });
}
